tr/help.md

```diff
@@ -63,6 +64,7 @@ In case an error is raised, make sure that the given expression can be correctly
 
 # Version History
 
+* 2.0.3 - Help updates
```

Comment:
We haven't released since last hackathon. Remove this line as we can release at 2.0.2.